### PR TITLE
jose/DisableLoginButton

### PR DESCRIPTION
**Pull request by Josetf2396**
If the email or password field have no input the login button is disabled

### CODE REVIEW

**Comment by Josetf2396** [comment]
Screen Recording 2022-04-06 at 8.23.35 PM

**Comment by Josetf2396** [comment]
https://user-images.githubusercontent.com/44705680/162114313-bee223e7-353d-4435-9df5-76954d817eb6.mov

**Comment by Josetf2396** [comment]
Screen-Recording-2022-04-06-at-8 23 35-PM

File: src/screens/login/LoginScreen.view.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ export const LoginScreenView = ({
           <ActivityIndicator size={'large'} color={color.primary} />
```
Comment (lasharela) [inline src/screens/login/LoginScreen.view.js:56]: We already have activity indicator in container. We don't need it here.

I'm from mobile but will take a look from laptop as well. Let's other folks to take a look

**Review by lasharela — CHANGES_REQUESTED** [review]

**Review by Dulaya — APPROVED** [review]

**Comment by Dulaya** [comment]
I remember Lasha saying to Charlie the ActivityIndicator from the React-Native library instead of creating a custom component.

**Comment by Dulaya** [comment]
For the ActivityIndicator, we can fix in this PR or make a new PR specifically for fixing ActivityIndicator. I noticed a lot of the screens use their own ActvityIndicator instead of the one from Container. @Josetf2396 Let me know if you need help.

**Review by leo-he-1 — APPROVED** [review]

**Review by carizzuti — APPROVED** [review]

File: src/screens/login/LoginScreen.view.js (hunk 1 above)
Comment (Dulaya) [reply src/screens/login/LoginScreen.view.js:56]: Our solution is to create a new story/PR for using the ActivityIndicator from Container for everything that we forgot.

**Pull request merged**